logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -529,8 +536,9 @@ internal class MLSConversationDataSource(
     override suspend fun removeMembersFromMLSGroup(
         groupID: GroupID,
         userIdList: List<UserId>
-    ): Either<CoreFailure, Unit> =
-        commitPendingProposals(groupID).flatMap {
+    ): Either<CoreFailure, Unit> = mutex.withLock {
+        kaliumLogger.d("cccc: removeMembersFromMLSGroup for ${groupID.toLogString()}")
```

Comment:
debug code to delete?

Reply:
My bad, I was trying to catch a deadlock